### DISCUSSION TITLE
Release Notes

### DISCUSSION THREAD

**Discussion by bordeo** [koush/scrypted#487] [Ideas]
Thank for this great platform. I really appreciate your work @koush.
I mainly use it for HomeKit Secure Video.

I'm writing to propose a system to write "Release Notes". Every time i found a new update available I would like to know what is changed, added or break so I can decide to update immediately or wait or just for the sake of reading it.

Currently I can find this information only on Github but without precision because tag is missing. Only npm packages and docker images are tagged.

Could you implement a "Release notes" system.

Thank you.

**Comment by jotala** [comment]
Totally agree. Even in the Discord, we have to digging to know if there are new features. 
I know this project is changing rapidly, but it would be really on the new major versions we have some clue on what’s is new.

**Comment by cheme75** [comment]
Agree, he seems to disregard those of us who prefer to track issues and keep up to date with GitHub which every other project developer seems to be happy to post release notes to keep users updated on the new features. Unfortunately, unless you post in discord, he probably won’t see this to respond.

Discord is pain since issues are all lumped together under one thread so there is no easy way to find solutions or answers. I only used scrypted to test how well I could use a few Wyze cams in HomeKit. But it’s such a poorly supported system, I’m not seeing much benefit for my needs versus just using the Wyze app and Alexa skill. The other Wyze devices work fine without much support with home bridge so I may just dump scrypted and go back to the camera ui plug-in.

**Comment by miwagner1** [comment]
![A66000D7-73E7-4388-89AC-4600E5184C1C](https://user-images.githubusercontent.com/4108500/222464704-b49bc1dc-b7a3-4eb6-b434-62e449ac0d78.png)
![371645BF-EC7D-4B05-B581-074849976994](https://user-images.githubusercontent.com/4108500/222464710-5658d961-2463-40a3-ab8b-c7b25db8eb28.png)

The tags are not missing? The list of comments are a change log? I use https://github.com/fmartinou/whats-up-docker and I have auto generated one tap links to see the changes.

Link example:
(https://github.com/koush/scrypted/releases/tag/v0.6.26)

**Reply by cheme75** [reply]
Will chk out. But seems like GitHub also has subscription options for notifications for followed projects. That seems to be enough to at least get notified by email.

**Comment by koush** [comment]
This was done a week ago. https://github.com/koush/scrypted/issues/569

**Reply by cheme75** [reply]
I just went thru my acct info to enable notification by email not just on GitHub. This should ensure updates. Thanks for being more available on GitHub. Early when first starting, it’s was difficult finding help here. Maybe it’s just taken some time for this to grow. 

Fwiw, I really do appreciate the work to develop this, even if it’s not all that necessary for my personal needs, I can see how it can benefit others.
